src/rasterizer/rasterizer.py

```diff
@@ -172,12 +172,26 @@ def __init__(self, resolution=256, steps=128, sigma=1e-2, method='base'):
 
         # TODO the follow line is required for the `tiled` method
         # self.gpu = torch.empty(2)
-        XX, YY = np.meshgrid(range(self.resolution), range(self.resolution))
+        left_pad = 0.25 * self.resolution
```

Comment:
Add a comment on where these #'s came from roughly. "Constants selected in according with empirical distribution function of point distributions from real fonts" w/e

Reply:
Done.